pinot-common/src/main/java/org/apache/pinot/common/function/AggregationFunctionType.java

```diff
@@ -30,6 +30,7 @@
   MIN("min"),
   MAX("max"),
   SUM("sum"),
+  SUMPRECISION("sumPrecision"),
```

Comment:
Why not allow both precision and scale?

Reply:
Yes, I was actually discussing this with @kishoreg 
I can allow both.

Reply:
done